lua/neogen/generator.lua

```diff
@@ -220,14 +220,14 @@ return setmetatable({}, {
             end
 
             -- Converts the content to a lsp compatible snippet
-            local generated_snippet, indent = snippet.to_snippet(content, marks_pos, { row, 0 })
+            local generated_snippet, col = snippet.to_snippet(content, marks_pos, { row, 0 })
```

Comment:
I still don't get why you want to remove indented spaces from the generated content, only to add them back afterwards with col. Why just avoid deleting spaces inside content, and append at col 0 ? 

Maybe I'm missing something here